hudi-cli/src/main/java/org/apache/hudi/cli/commands/RepairsCommand.java

```diff
@@ -18,7 +18,7 @@
 
 package org.apache.hudi.cli.commands;
 
-import org.apache.hudi.cli.DeDupeType;
+import org.apache.spark.sql.hudi.DeDupeType;
 import org.apache.hudi.cli.HoodieCLI;
```

Comment:
The hudi-cli-bundle does not really include the hudi-spark-xxx jars, do we need to package them all together?

Reply:
Thank you for helping to review the code, ！ found some duplicate code, I want to improve this part of the code. Do we need to keep `org.apache.hudi.cli.DedupeSparkJob` ? and replace `org.apache.spark.sql.hudi.DedupeSparkJob`, I want to hear your opinion.

Reply:
Need to ping the author why there are duplciates here, cc @nsivabalan do you have any impression why these classes are duplicated here?